docker-builds/reference-crucible/Dockerfile

```diff
@@ -19,6 +19,10 @@ RUN apt-get install -y mongodb mongodb-server && \
 RUN mkdir -p /etc/service/mongodb/
 ADD mongodb.sh /etc/service/mongodb/run
 
+# Set up delayed jobs
+ADD delayed-job /etc/init.d/delayed-job
+RUN update-rc.d delayed-job defaults
```

Comment:
Yeah, it looks like that doesn't actually get started, I was just copying the crucible install instructions. (Starting it unfortunately doesn't fix our "can't run tests" problem though)